### DISCUSSION TITLE
Multiple nucleation conditions

### DISCUSSION THREAD

**Discussion by ashishdhole** [idaholab/moose#28884] [Q&A Modules: Phase field]
Hello,
I am working on phase transformation and need nucleation event happening at different region with different conditions. Can I use the following:
```
[UserObjects]
  [inserter1]
    type = DiscreteNucleationInserter
    hold_time = 0.4
    probability = P
    seed = 12346
    radius = 3.27
  []
  [inserter2]
    type = DiscreteNucleationFromFile
    hold_time = 10
    file = nuclei2.csv
    radius = 2
  []
  [map1]
    type = DiscreteNucleationMap
    int_width = 2
    periodic = eta1
    inserter = inserter1
  []
  [map2]
    type = DiscreteNucleationMap
    int_width = 2
    periodic = eta2
    inserter = inserter2
  []
[]
```
or is there any other way?
Thank you

**Comment by GiudGiud** [comment]
Hello

The documentation for this technique can be found there
https://mooseframework.inl.gov/modules/phase_field/Nucleation/DiscreteNucleation.html

I dont know if we have other methods in moose

Guillaume
